aboutme.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="./css/aboutme.css">
+    <title>togoville jazz festival aboutme</title>
+</head>
+<body>
+    <header>
+        <ul class="header-links">
+            <li><a href="#" class="social-link fa fa-twitter"></a></li>
+            <li><a href="#" class="social-link fa fa-facebook"></a></li>
+            <li><a class="lato-font-black" href="#">English</a></li>
+            <li><a class="lato-font-black" href="#">My page</a></li>
+            <li><a class="lato-font-black" href="#">Logout</a></li>
+        </ul>
+        <nav class="nav-bar">
+            <div class="hamburger-class">
+                <a href="#" ><img src="./img/iconmonstr-menu-1-240.png" alt="menu button"></a>
+            </div>
+            <ul class="menu-bar">
+                <li class="menu-logo" ><a href="./index.html"><img src="./img/level production icon 3.png" alt="level production icon"></a></li>
+                <li><a class="lato-font-black aboutme-link" href="./aboutme.html">About</a></li>
```

Comment:
- [optional] I think it will be more professional to have **homepage link** in About page, not **aboutpage link** in About page. Kindly implement this for both mobile and desktop versions. 👍

Reply:
Hi @OyePriscilla,  
Thank you for your review.
Please do I need to replace the "About" with "Home page" in the nav bar when I will be on the about me page? 
Please note that my logo is already linked to the home page in line 25 of aboutme.html

Reply:
@Matthieu96Code, yes please! This will enhance the user experience. 👍